src/Frontend/Path.hs

```diff
@@ -25,7 +25,7 @@ module Frontend.Path
     ( HasPath(..)
     , Top(..)
     , AllowedMethod(..)
-    , Main
+    , Main(Broken)
```

Comment:
why?  won't do it!  (-:

i don't see any benefit (it's called "broken" and shouldn't even be locally defined in this module, and i was confused by the fact it was hidden and have wasted some time figuring out why.)